furusiya/main_interface.py

```diff
@@ -25,7 +25,7 @@ class Game:
     game_msgs = []
     game_state = None
 
-    area_map: AreaMap = None
```

Comment:
Why did you remove this? It helps with auto completion :/

Reply:
I removed it because it makes the game crash. I thought I replaced it with `area_map = None`.